dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/config/ServiceConfig.java

```diff
@@ -255,6 +255,7 @@ public RetryTemplate retryTemplate()
     public Map<ImportStrategy, List<Class<? extends Processor>>> eventInsertPostProcessorMap()
     {
         return ImmutableMap.of( CREATE, newArrayList(
+            ProgramNotificationPostProcessor.class,
```

Comment:
Can we move the `ProgramNotificationPostProcessor` class to `shared/postprocess` package?

Reply:
Yep, we already talked about it, I forgot :+1: